int16[] aruco_ids
sensor_msgs/NavSatFix gps_coord
---
sensor_msgs/NavSatFix optimal_gps_coord
bool use_optimal
---
